1_fetch/src/my_happy_script.R

```diff
@@ -51,7 +52,7 @@ eval_data <- function(in_csv){
 
 
 # Create a plot, this function won't return anything
-createPlot <- function(eval_data){
+createPlot <- function(evalData){
```

Comment:
this could have an `out_folder` argument as well that you can set to `3_visualize/out` or whatever you want to call the 3 step.